### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by llhouse2**
Check the .csv file for the subcategories.

### CODE REVIEW

File: cropScapeDocumentation/CDL_subcategories.csv [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+0,Background,N,N,,,,,,,,,
+1,Corn,A,A,,,,,,,,,
+2,Cotton,A,A,,,,,,,,,
+3,Rice,A,A,,,,,,,,,
+4,Sorghum,A,A,,,,,,,,,
+5,Soybeans,A,A,,,,,,,,,
+6,Sunflower,A,A,,,,,,,,,
+7, ,N ,N ,,,,,,,,,
+8, ,N ,N ,,,,,,,,,
+9, ,N ,N ,,,,,,,,,
```
Comment (johnbradley) [inline cropScapeDocumentation/CDL_subcategories.csv:11]: Why are there seemingly empty rows?

Comment (johnbradley) [reply cropScapeDocumentation/CDL_subcategories.csv:11]: Should I just ignore them?

Comment (llhouse2) [reply cropScapeDocumentation/CDL_subcategories.csv:11]: I will re-do... hold tight.

Comment (llhouse2) [reply cropScapeDocumentation/CDL_subcategories.csv]: All done.

**Review by johnbradley — APPROVED** [review]
LGTM

**Pull request merged**